test/lib/create-next-install.js

```diff
@@ -37,7 +37,7 @@ async function createNextInstall(
     }
   }
 
-  for (const item of ['package.json', 'yarn.lock', 'packages']) {
+  for (const item of ['package.json', 'packages']) {
```

Comment:
Do we need the pnpm lockfile here?

Reply:
The lockfile wasn't actually be used when copied here so I think we were just copying it for brevity